templates/web/oxfordshire/admin/report_edit_extra_info.html

```diff
@@ -0,0 +1,12 @@
+<li class="sm">
+    [% loc('RDI sent:') %]
+    [% IF problem.inspection_log_entry %]
+        [% IF problem.get_extra_metadata('rdi_processed') %]
```

Comment:
If this is true, the former must be true, so can swap and not have to do that check?
It also looks like we set 'inspected' on problem metadata (used elsewhere in template), which presumably saves an admin log search if we used that.